src/test/scala/net/mikolak/travesty/VizGraphProcessorSpec.scala

```diff
@@ -101,6 +101,17 @@ class VizGraphProcessorSpec extends FlatSpec with MustMatchers with MustVerb {
       .map(_.toString)
   }
 
+  it must "support unnamed stages" in {
+    val input = Source.single("hello")
+      .log("logger")
+      .to(Sink.head)
+
+    val output = calculateResult(input)
+
+    output.nodeList() must have size 3
+    output.linkList() must have size 2
```

Comment:
Once you add the configurable default name, there should be an assertion that checks whether that name is used as well.